### ISSUE TITLE
Video becomes black once I return back to video after scrolling. 

### ISSUE THREAD

**Issue by AniketBito** [eneim/toro#242]
What kind of issue is this?

- [ ] Question. If so, declare clearly your **target** and your current implementation with code formatted in markdown.

- [ ] Bug report. If so, please supply the way to reproduce the issue, including addition information as follow:
  - [ ] Toro version.
  - [ ] Android version (can be found in Device info). **Rooted/Custom rom devices are not supported**.
  - [ ] Device name (eg: Nexus 5X, Samsung Galaxy S8)
  - [ ] Source code that can reproduce the issue (a snapshot is expected).
  - [ ] Logcat stack trace in case of fatal issue.
  
- [ ] Feature request. Start by telling me what problem you’re trying to solve. Don’t send pull requests to implement new features without first getting my support. Sometimes I leave features out on purpose to keep the project small.

- [ ] Other. Senseless issues are normally ignored. Please understand and cooperate.

**Comment by eneim** [comment]
@AniketBito In that case, please provide more information. Also if possible please help to give me a snapshot of code that doesn't work for you if possible (using dummy video/big buck bunny for your content as well).

**Comment by AniketBito** [comment]
@eneim please check this class. This is the code which I have used
[Java Class.docx](https://github.com/eneim/toro/files/1195957/Java.Class.docx)

**Comment by AniketBito** [comment]
Ideally the video should stop the the focus is gone and should resume when I come back but this doesn't happen

**Comment by eneim** [comment]
@AniketBito Thanks. What do you mean by "the focus is gone" (the actual Android system event or UI event?). Also your comment doesn't match your issue title. Can you clarify about the following thing:

- What is your expected behaviour using your class above?
- What is actually happening?

Also please avoid setting toroStrategy from ViewHolder.

**Comment by AniketBito** [comment]
Focus is gone mean when I scroll the video up and the video moves out of the screen.
Expected behaviour :  When the scroll up and if the video moves out of the screen then the video should stop or pause and once I scroll down to come back to the same video then it should start from the very same point.

What is happening : When I come back the next time video is showing me black screen and until and I need to forcefully start the video on onPlaybackPaused and then the video resumes.
But this takes around 5-10 seconds and till that time the black screen appears

**Comment by eneim** [comment]
@AniketBito Ok, it is more clear for me. About the focus thing, the reason I ask is a ViewGroup will have a focused child event though the un-focused children is still visible, so I want to make sure we are on the same page. Let me investigate your case.

Also I understand that migration is hard, but if you need my help I can provide a few advice to move from ver 2 to ver 3 :D.

**Comment by AniketBito** [comment]
Please try to help me with this case first.

Then if there seems no solution then we will think of moving to ver 3

**Comment by eneim** [comment]
@AniketBito Of course.

One question: ```return (ExoVideoView) itemView.findViewById(R.id.btnRecord);``` <-- Is ```R.id.btnRecord``` is the Id for the VideoView? (Just be curious).

**Comment by AniketBito** [comment]
Yes. It's the ID of exovideo view(im.ene.toro.exoplayer2.ExoVideoView)

**Comment by eneim** [comment]
@AniketBito Also I see that you are manually starting the playback in onPlaybackComplete. Do you want to loop the player?

**Comment by AniketBito** [comment]
Yes.

**Comment by eneim** [comment]
@AniketBito I think a will have a few question, if you feel not convenience here in issue, we can move to ```https://gitter.im/eneim/Toro``` (the chat channel for this repo) for better responsive :D.

My next question is which specific version of Toro are you using now? If it is not 2.3.0 (latest v2 release), don't you mind migrate to 2.3.0?

**Comment by AniketBito** [comment]
Yeah sure why not.

Let's move there

**Comment by pro-softs** [comment]
@AniketBito Hi, I am facing the same black screen problem on scrolling. How did u solved the issue??
